be2-scala/src/test/scala/ch/epfl/pop/model/network/method/message/data/socialMedia/AddReactionSuite.scala

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.pop.model.network.method.message.data.socialMedia
+
+import org.scalatest.{FunSuite, Matchers}
+import ch.epfl.pop.model.objects.Timestamp
+import ch.epfl.pop.json.MessageDataProtocol._
+import ch.epfl.pop.model.network.method.message.data.{ActionType, ObjectType}
+import spray.json._
+
+class AddReactionSuite extends FunSuite with Matchers {
```

Comment:
Same remark:
Test suite would look better if you put `AddReaction` messages creation in a separate scala object (same file should be fine)

Reply:
same as above